packages/reference/environments/production/us-east-2/authentik_mongodb_atlas_sso/terragrunt.hcl

```diff
@@ -0,0 +1,28 @@
+include "panfactum" {
+  path   = find_in_parent_folders("panfactum.hcl")
+  expose = true
+}
+
+terraform {
+  source = include.panfactum.locals.pf_stack_source
+}
+
+dependency "authentik_core" {
+  config_path = "../authentik_core_resources"
+}
+
+dependency "kube_authentik" {
+  config_path = "../kube_authentik"
+}
+
+inputs = {
+  acs_url     = "https://auth.mongodb.com/sso/saml2/0oaw7vqdsehzxtqZ1297"
+  audience    = "https://www.okta.com/saml2/service-provider/spzsbkposqvrzhbjcdnz"
```

Comment:
If this is always `https://www.okta.com/saml2/service-provider`, can we just have `spzsbkposqvrzhbjcdnz` or does it make more sense to always have the full string? Your call.

Reply:
I'm going to leave the full string as I have no guarentee the url format won't change in the future. Also think it would be more more confusing to the user if we have to explain just get the id portion.